src/batch/client.ts

```diff
@@ -149,37 +132,30 @@ export class BatchTranscription {
       15 * 60 * 1000, // 15 minutes timeout
     );
 
-    return await this.getJobResult(submitResponse.id, format);
+    return await this.getJobResult(submitResponse.id, format || 'json-v2');
```

Comment:
perhaps getJobResult should accept undefined for the second argument and the default be declared there

Reply:
True, I think at the moment the overloading of the argument types requires it to be defined, but let me see if I can achieve the same behaviour with it as an optional argument